backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/question/domain/MultipleChoiceQuestion.java

```diff
@@ -165,4 +149,9 @@ public String toString() {
                 "options=" + options +
                 '}';
     }
+
+    public static String convertSequenceToLetter(Integer correctAnswer) {
+        return Character.toString('A' + correctAnswer);
```

Comment:
I did this change, but what do you think if instead of 'X' having '-' for not answered questions? think it would be interesting to not be confused with eventual option "X"🤔

Reply:
Yes, "-" is a better solution!